src/modules/dashboard/components/ActionsPage/FinalizeMotionAndClaimWidget/FinalizeMotionAndClaimWidget.tsx

```diff
@@ -100,6 +100,10 @@ export const MSG = defineMessages({
     id: 'dashboard.ActionsPage.FinalizeMotionAndClaimWidget.winningsLabel',
     defaultMessage: `Winnings`,
   },
+  penaltyLabel: {
+    id: 'dashboard.ActionsPage.FinalizeMotionAndClaimWidget.penaltyLabel',
+    defaultMessage: `Penalty`,
+  },
```

Comment:
I would have just combined the two message descriptors into one, and use the `select` declaration and pass a `value` to determine which one to show, but I'm good wither way.